coccinelle/glnx-errors.cocci

```diff
@@ -0,0 +1,6 @@
+@@
+expression p;
+@@
+- glnx_set_error_from_errno (p);
+- goto out;
++ return  glnx_throw_errno (p);
```

Comment:
What I mean is, this specific patch has no practical use for now, right? (Exactly because of what we mentioned above). I.e. this is something that we'd need to work on before we can apply it on the whole codebase.